templates/vue/src/components/tyde/TydeModule.vue

```diff
@@ -54,9 +54,11 @@ export default {
     ...mapMutations(["openLightbox", "closeLightbox"]),
     openLightbox(id) {
       this.$router.push(`/nodes/${id}`)
+      document.querySelector("body").style.overflow = "hidden"
     },
     closeLightbox() {
       this.$router.push(`/`)
+      document.querySelector("body").style.overflow = "auto"
```

Comment:
This is not working because it's attached to the lightbox which is different than a stage.